MediaBrowser.LocalMetadata/Images/LocalImageProvider.cs

```diff
@@ -218,7 +218,8 @@ private void PopulateImages(BaseItem item, List<LocalImageInfo> images, List<Fil
             "folder",
             "poster",
             "cover",
-            "default"
+            "front",
+            "default",
```

Comment:
```suggestion
            "default"
```

Reply:
It's actually recommended to have the coma - https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1413.md
Do you want it removed anyway?

Reply:
That would be a question for @Bond-009, what do you think?

Reply:
> It's actually recommended to have the coma - https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1413.md
Do you want it removed anyway?

It isn't https://github.com/jellyfin/jellyfin/blob/master/jellyfin.ruleset#L26